CHANGELOG.md

```diff
@@ -287,6 +287,9 @@ _Code:_
 
   - New class `SymmetricKey` can be used to generate symmetric keys for Secure Cell ([#562](https://github.com/cossacklabs/themis/pull/562)).
   - New makefile target `make jsthemis` can be used to build JsThemis from source ([#618](https://github.com/cossacklabs/themis/pull/618)).
+  - `SecureCell` now allows `null` to explicitly specify omitted encryption context ([#620](https://github.com/cossacklabs/themis/pull/620)).
+  - `SecureMessage` now allows `null` for omitted keys in sign/verify mode ([#620](https://github.com/cossacklabs/themis/pull/620)).
+  - Fixed a crash when an exception is thrown from `SecureSession` callback ([#620](https://github.com/cossacklabs/themis/pull/620)).
```

Comment:
shouldn't we add that we've changed the type of thrown errors from `NaN::Error` to `NaN::TypeError`? I'm not sure how js devs are used to catch errors (if they do catch errors), but it might be useful for them to know that error type has changed.

Reply:
> shouldn't we add that we've changed the type of thrown errors?

I don't think it's necessary. It is a programming error to pass an invalid type so it should be caught during development, not at run-time. If it's absolutely necessary to catch it at runtime then any existing catch clause for `Error` (which we have been throwing before) will catch `TypeError` too because it's a subclass.

Reply:
okay, makes sense